cache/clock_cache.h

```diff
@@ -614,10 +621,18 @@ class FixedHyperClockTable : public BaseClockTable {
   const std::unique_ptr<HandleImpl[]> array_;
 };  // class FixedHyperClockTable
 
+// Placeholder for future automatic table variant
+// For now, just use FixedHyperClockTable.
+class AutoHyperClockTable : public FixedHyperClockTable {
+ public:
+  using FixedHyperClockTable::FixedHyperClockTable;
+};  // class AutoHyperClockTable
+
 // A single shard of sharded cache.
-template <class Table>
```

Comment:
Just for my own learning, is this change and below "using... " a functional change? If so, I wonder why it doesn't work as is.

Reply:
lru_cache_test uses the Table type, and this change makes it a public member type of the Shard type, so the test can use Shard::Table.